deploy/kubernetes/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Not sure we need this, as above.

Reply:
Having a dedicated namespace is good practice.
Picking "cvmfs" as the namespace name is an admin choice, although it does seem an obvious one.

Reply:
We're running all the system stuff in kube-system, including cvmfs.

Reply:
As I said, not essential. 
No objections from my part to remove the use of a dedicated namespace from the CERN git repo.